src/animationLoop.js

```diff
@@ -86,7 +86,7 @@ export default function configAnimation(config = {}) {
 
   return function startAnimation(state, animationStep, animationRender) {
     for (let i = 0; i < animRunning.length; i++) {
-      let val = animRunning[i];
+      const val = animRunning[i];
```

Comment:
I’ll suppress the `prefer-const` rule then, unless anyone knows about a way to enforce this particular coding style. We should propose a new rule to ESLint, because it’s a common convention.